app/components/sign-up/signup-form.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="user-details">
+  <h2 class="user-details__heading">
+    {{this.label}}
+  </h2>
+  <div class="user-details__field">
+    <Input @value={{get @userDetails @state}} type="text" placeholder="Darth" {{on 'input' (fn this.inputFieldChanged)}} />
```

Comment:
NIT: If we're not passing an argument, we don't need the (fn) helper, just thought I'd mention it for learning purposes :)

Reply:
Cool, Thanks :)